targets/unity-v2/make.js

```diff
@@ -431,7 +436,14 @@ function getRequestActions(tabbing, apiCall, isApiInstance = false) {
 	if (apiCall.name === "GetEntityToken" && isApiInstance === true)
         return tabbing + "AuthType authType = AuthType.None;\n" +
             "#if !DISABLE_PLAYFABCLIENT_API\n" +
-            tabbing + "if (authType == AuthType.None && !string.IsNullOrEmpty(request.AuthenticationContext.ClientSessionTicket ?? (authenticationContext.ClientSessionTicket ?? PluginManager.GetPlugin<IPlayFabTransportPlugin>(PluginContract.PlayFab_Transport).AuthKey)))\n" +
+            tabbing + "if (request.AuthenticationContext != null && !string.IsNullOrEmpty(request.AuthenticationContext.ClientSessionTicket))\n" +
+            tabbing + "    clientSessionTicket = request.AuthenticationContext.ClientSessionTicket;\n" +
+            tabbing + "if (clientSessionTicket == null && authenticationContext != null && !string.IsNullOrEmpty(authenticationContext.ClientSessionTicket))\n" +
+            tabbing + "string clientSessionTicket = null;\n" +
```

Comment:
>tabbing + "string clientSessionTicket = null;\n" + [](start = 12, length = 50)

This line with declaration:
string clientSessionTicket = null
should be moved 3 lines above, right?

Reply:
yes you're right it moved below accidently